extensions/mssql/src/errorDiagnostics/errorDiagnosticsConstants.ts

```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as nls from 'vscode-nls';
+const localize = nls.loadMessageBundle();
+
+// Error code reference comes from here: https://learn.microsoft.com/en-us/sql/relational-databases/errors-events/database-engine-events-and-errors?view=sql-server-ver16
+export const MssqlPasswordResetErrorCode: number = 18488;
+
+export const MssqlDiagnosticsProviderDisplayName = localize('mssql.diagnosticsProviderDisplayName', "Azure SQL Diagnostics for MSSQL");
```

Comment:
I don't think this needs to be localized as this is not user facing.

Reply:
Why is this property called "displayName" then if it's not user facing? Do we know where this is even used? It feels like it's not even a property we should have for a feature like this (except perhaps for logging purposes, but the ID would do fine for that I think)

Reply:
I took a look at the service side code, only one diagnostics provider is allowed for a connection provider, based on that, we shouldn't need to name the diagnostics provider.

Reply:
It depends on the context, you could definitely want to have a readable "display name" even if there's only one registered per provider of course. 

But yeah, in this context it seems weird and unnecessary to have displayName even be an option. Simpler to just remove it.